doc/changelog.md

```diff
@@ -4,6 +4,30 @@
 
 <h3>Improvements</h3>
 
+* Catalyst now supports return statements inside conditionals in `@qjit(autograph=True)` compiled
+  functions.
+  [(#)]()
+
+  The following is now possible:
+
+  ```py
+  @qjit(autograph=True)
```

Comment:
I wonder why the syntax highlighting is not working 🤔

Reply:
On GitHub I'm not sure, it doesn't seem to syntax highlight code blocks in the code view of markdown files. It is highlight in the rendered version on GitHub or the docs though.